dallinger/recruiters.py

```diff
@@ -1,9 +1,8 @@
 """Recruiters manage the flow of participants to the experiment."""
-
 import os
 
 from boto.mturk.connection import MTurkConnection
-from psiturk.models import Participant
+from dallinger.models import Participant
```

Comment:
No, I think that was the only motivation. I may have run into a problem with the original import statement that I was trying to side-step(?)